demos/c_api_minimal_app/README.md

```diff
@@ -130,11 +130,13 @@ Usage:
                                 INFO, WARNING, ERROR (default: ERROR)
       --config_path CONFIG_PATH
                                 Config file path for OVMS to read (default:
-                                /ovms/src/test/c_api/config_benchmark.json)
+                                /ovms/src/test/configs/config_benchmark.json)
       --niter NITER             number of inferences to conduct (default:
                                 1000)
-      --nstreams NSTREAMS       nstreams from OVMS configuration (default:
-                                1)
+      --nstreams NSTREAMS       number of execution streams to be performed
+                                simultaneously (suggested for best
+                                throughput is NUM_STREAMS of a ovms model
+                                config) (default: 1)
```

Comment:
`(suggested for best throughput is NUM_STREAMS of a ovms model config)`
This part sounds a bit confusing. After reading that part I probably still wouldn't know what value I should put there for best throughput.